Sources/App/Commands/TriggerBuilds.swift

```diff
@@ -78,16 +78,21 @@ struct TriggerBuildsCommand: Command {
                 throw UsageError()
         }
 
-        try triggerBuilds(on: context.application.db,
-                          client: context.application.client,
-                          logger: logger,
-                          mode: mode).wait()
+        do {
+            try triggerBuilds(on: context.application.db,
+                              client: context.application.client,
+                              logger: logger,
+                              mode: mode).wait()
+        } catch {
+            logger.critical("\(error)")
```

Comment:
I'm never sure whether to use just `"\(error)"` or `"\(error.localizedDescription)"` but when testing this it's clear that the former has the better info:

```
[ CRITICAL ] keyNotFound(CodingKeys(stringValue: "web_url", intValue: nil), Swift.DecodingError.Context(codingPath: [], debugDescription: "No value associated with key CodingKeys(stringValue: \"web_url\", intValue: nil) (\"web_url\").", underlyingError: nil)) [component: trigger-builds]
```

vs

```
[ CRITICAL ] The data couldn’t be read because it is missing. [component: trigger-builds]
```